include/summary-4.tex

```diff
@@ -18,9 +18,9 @@
 to define an element of~$X$ which, by the very way we define it, is
 guaranteed to be different from every element in the enumeration. So
 the enumeration can't be an enumeration of all of~$X$ after all, and
-we've shown that no enumeration of~$X$ can exist.  A reduction shows
+we've shown that no enumeration of~$X$ can exist. A reduction shows
 that $X$ is uncountable by associating every element of some known
-uncountable set~$Y$ with an element of~$X$ in a bijective way. If this
+uncountable set~$Y$ with an element of~$X$ in an injective way. If this
```

Comment:
It is correct, but not in line with the section being summarized (although neither was the old version). It should be "by associating every element of~$X$ with an element of some known uncountable set~$Y$ in a surjective way."